test/perf/perf_fast_forward.cc

```diff
@@ -1885,7 +1885,8 @@ namespace perf {
 
 int scylla_fast_forward_main(int argc, char** argv) {
     namespace bpo = boost::program_options;
-    app.add_options()
+    app = new app_template();
```

Comment:
This will generate leak reports in debug mode.
I think a much better change is to remove the global `app` altogether, put it on the stack and just propagate it to those 2 functions that need it.

Reply:
Put `app` on the stack and added `app_template& app` for the function argument.
Changes become bit large since `on_test_case()` isn't directly called from `scylla_fast_forward_main()`, so we have to add `app` on all related functions.